src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/RdfViewConfiguration.java

```diff
@@ -0,0 +1,73 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper;
+
+import java.util.Set;
+
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.GraphQLEntrypoint;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLEntrypointType;
+
+/**
+ * Used to represent and retrieve information about a virtual RDF view for a GraphQL endpoint
+ */
+public interface RdfViewConfiguration {
+
+    /**
+     * Verify that @param className exist
+     */
+    public boolean containsClass(String className);
+
+    /**
+     * Verify that @param uri that correponds to a class exists
+     */
+    public boolean containsClassURI(String uri);
+
+    /**
+     * Verify that @param propertyName of @param className exist
+     */    
+    public boolean containsProperty(String className, String propertyName);
+
+    /**
+     * Verify that @param uri that corresponds to a property exist for @param className
+     */
+    public boolean containsPropertyURI(String className, String uri);
+
+    /**
+     * Verify that @param property of @param className links to a nested object
+     */
+    public boolean isObjectProperty(String className, String propertyName);
```

Comment:
does `containsProperty` refer to datatype properties? why do we need a different method for both types of properties?